<launch>
  <node name="usb_cam" pkg="usb_cam" type="usb_cam_node" output="screen" >
    <param name="video_device" value="/dev/CAM0" />
    <param name="image_width" value="640" />
    <param name="image_height" value="480" />
    <param name="pixel_format" value="yuyv" />
    <param name="color_format" value="yuv422p" />
    <param name="camera_frame_id" value="usb_cam" />
    <param name="io_method" value="mmap"/>
  </node>

  <node name="OpencvServer" pkg="opencv_server" type="opencv_server_node" respawn = "true" output="screen" >
    <param name="/pic_save_path" value="/home/tspi/" />
    <param name="/QR_file_name" value="newst.jpg" />
    <param name="/use_window" type="bool" value="true" />
  </node>
  <!-- <node name="image_view" pkg="image_view" type="image_view" respawn="false" output="screen">
    <remap from="image" to="/usb_cam/image_raw"/>
    <param name="autosize" value="true" />
  </node> -->
</launch>
